### ISSUE TITLE
Issue with Storage Drawers compacting drawers 

### ISSUE THREAD

**Issue by Tuchevery** [Lothrazar/Storage-Network#173]
Minecraft version & Mod Version:
1.12.2, 1.7.4

Single player or Server:
single

Describe problem (what you were doing / what happened):
I'm using storage network with version 5.3.8 of the storage drawers mod.  I can't see the contents of the compacting drawers from a distance using any of the unlimited distance remotes.  I can see them when close by.  

BTW, I love your mod!

Crash log (if any):
none

(If this is a suggestion and not a bug report, delete all this and just write your suggestion!)

**Comment by Tuchevery** [comment]
I disconnected the compacting drawers from the storage drawers controller and ran storage link cables directly to them.  It did not fix the issue.  I can see everything using the remote when I am close to them but I cannot see their contents when I am away (400 blocks).  I suspect I lose their contents after 64 blocks but I need to confirm that.

**Comment by thraaawn** [comment]
Did you chunk load the storage drawers?
If that doesn't help, I'm afraid you might have to chunkload all cables between ssn-master and the drawers.

**Comment by Tuchevery** [comment]
thraawn you were right, thank you!

At first I didn't think it was a chunk issue because I could see some of the items regardless of distance.  I showed the chunk borders and sure enough the drawers I couldn't see at distance were drawers in the chunk next to the one with the drawer controller and network controller.    

I assumed if it was a chunk loading issue I wouldn't be able to see any drawers at distance but it looks like the drawer controller and/or network controller keep the chunk they are in loaded all the time.

I solved the issue by installing a chunk loader from chicken chunks.

Thanks again thraawn!
